pkg/config/sync.go

```diff
@@ -276,6 +290,14 @@ func logCurrentConfig() {
 			config = append(config, fmt.Sprintf("workspace.ignoredUnrecoverableEvents=%s",
 				strings.Join(Workspace.IgnoredUnrecoverableEvents, ";")))
 		}
+		if Workspace.DefaultStorageSize != nil {
+			if Workspace.DefaultStorageSize.Common != nil {
+				config = append(config, fmt.Sprintf("workspace.storageClassSize.%s=%s", constants.CommonStorageClassType, Workspace.DefaultStorageSize.Common.String()))
+			}
+			if Workspace.DefaultStorageSize.PerWorkspace != nil {
+				config = append(config, fmt.Sprintf("workspace.storageClassSize.%s=%s", constants.PerWorkspaceStorageClassType, Workspace.DefaultStorageSize.PerWorkspace.String()))
```

Comment:
Should now be: `"workspace.defaultStorageSize.%s=%s"` instead of `"workspace.storageClassSize.%s=%s"`